test/intl402/Segmenter/constructor/constructor/options-null.js

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2019 Leo Balter. All rights reserved.
```

Comment:
Your name is not Leo Balter ;)

Reply:
I copied a file author by him and modify from there. I don't think I should change his original copyright from that, right?